diff --git a/node_modules/@hyperledger/aries-askar-nodejs/build/NodeJSAriesAskar.js b/node_modules/@hyperledger/aries-askar-nodejs/build/NodeJSAriesAskar.js
index 0211aeb..3af77ed 100644
--- a/node_modules/@hyperledger/aries-askar-nodejs/build/NodeJSAriesAskar.js
+++ b/node_modules/@hyperledger/aries-askar-nodejs/build/NodeJSAriesAskar.js
@@ -528,7 +528,11 @@ class NodeJSAriesAskar {
     }
     async storeOpen(options) {
         const { profile, keyMethod, passKey, specUri } = (0, ffi_1.serializeArguments)(options);
-        const handle = await this.promisifyWithResponse((cb, cbId) => this.nativeAriesAskar.askar_store_open(specUri, keyMethod, passKey, profile, cb, cbId), ffi_1.FFI_STORE_HANDLE);
+        const handle = await this.promisifyWithResponse((cb, cbId) => {
+          const y = this.nativeAriesAskar.askar_store_open(specUri, keyMethod, passKey, profile, cb, cbId) 
+          this.handleError()
+          return y
+        }, ffi_1.FFI_STORE_HANDLE);
         return aries_askar_shared_1.StoreHandle.fromHandle(handle);
     }
     async storeProvision(options) {
